packages/web/src/exporters/otlp.ts

```diff
@@ -61,7 +63,8 @@ export class SplunkOTLPTraceExporter extends OTLPTraceExporter {
         // need to test with actual ingest
         Accept: 'application/json',
         'Content-Type': 'application/json',
-        ...this.headers
+        Authorization: this.apiToken,
```

Comment:
is this required?

Reply:
Yeah it is required to set the api key, I tried to find other options, but didn't work.

Reply:
let's use the same name in all places `rumAccessToken`